openwisp_users/tests/test_admin.py

```diff
@@ -256,3 +256,24 @@ def test_admin_add_superuser_org_not_required(self):
         user = queryset.first()
         self.assertTrue(user.is_superuser)
         self.assertTrue(user.is_staff)
+
+    # def test_operator_change_user_permissions(self):
+    #     operator = self._create_operator()
+    #     operator.is_staff = True
```

Comment:
you can move this inside the definition of `_create_operator` in [tests/utils.py](https://github.com/openwisp/openwisp-users/blob/master/openwisp_users/tests/utils.py) as it makes sense to have it there